ci/prow/config.yaml

```diff
@@ -1651,8 +1354,6 @@ periodics:
   name: ci-knative-serving-go-coverage
   agent: kubernetes
   decorate: true
-  ssh_key_secrets:
-  - ssh-knative
   extra_refs:
   - org: knative
     repo: serving
```

Comment:
below this there is the perf cron, which also needs to be updated. I cant comment on that line :)

Reply:
That's weird, my local copy was updated and had no ssh entries anymore... Thanks for pointing it out, let me fix it again.